### ISSUE TITLE
Getting errors with storing date to memory using textgen API

### ISSUE THREAD

**Issue by oderwat** [cpacker/MemGPT#162]
I wonder if this is a problem with my setup and if I can fix it (and how):

```
FUNCTION RETURN: {"status": "OK", "message": null, "time": "2023-10-28 05:56:00 AM PDT-0700"}
USER: Hi, my name is hans, please store this in your memory
ASSISTANT:
{
  "function": "core_memory_append",
  "params": {
    "inner_thoughts": "Storing new information about the user. The human's name is Hans.",
    "name": "human",
    "content": "First name: Hans"
  }
}
FUNCTION RETURN: {"status": "Failed", "message": "Error calling function core_memory_append with args {'name': 'human', 'content': 'First name: Hans'}: object NoneType can't be used in 'await' expression", "time": "2023-10-28 05:56:16 AM PDT-0700"}
USER:
    "inner_thoughts": "Encountered an unexpected error while processing the user's request. Trying to recover...",
    "message": "Apologies, it seems there was a temporary issue with updating your details. Please bear with me while I resolve this."
  }
}
```

**Comment by elijahsmith** [comment]
+1, seems to have been introduced in this commit https://github.com/cpacker/MemGPT/commit/45692f4f8addebc264da1e38d172aab892d047db

**Comment by cpacker** [comment]
> +1, seems to have been introduced in this commit [45692f4](https://github.com/cpacker/MemGPT/commit/45692f4f8addebc264da1e38d172aab892d047db)

Thanks for catching this, looking into it and will patch ASAP.

**Comment by cpacker** [comment]
This should be fixed in the latest package (0.1.8):

<img width="967" alt="image" src="https://github.com/cpacker/MemGPT/assets/5475622/f595cfc5-230f-4490-aea0-186abd702b33">

```sh
$ pip list | grep pymemgpt
pymemgpt                      0.1.8
```

Please reopen if you run into the same issue again! Thank you for the bug reports 🙏
